src/components/ExecutorDetails/ExecutorDetails.js

```diff
@@ -46,25 +45,34 @@ import columns from './Columns';
 import { useGetEntities } from './helpers';
 
 let refreshInterval;
+const _ = require('lodash');
```

Comment:
Since we only need `isEmpty` we could use `import isEmpty from 'lodash/isEmpty'`.